services/components/add-component-handler.js

```diff
@@ -133,6 +133,8 @@ function addHandler(button, options, prevRef) {
 
 module.exports = addHandler;
 module.exports.getParentEditableElement = getParentEditableElement;
+_.set(window, 'kiln.services.getAddableComponents', getAddableComponents); // export for plugins
```

Comment:
Hmm I haven't needed `getParentEditableElement` so I didn't want to expose something that might never be used. But if we want to expose it I'm down.